Docs and README update for ZenML Cloud (#1723)
zenml-io/zenml: docs/book/deploying-zenml/zenml-cloud/get-started.md

---
description: Your first steps to join ZenML Cloud and set up your first tenant.
---

# Get started

This page will walk you through the simple steps to sign up for the ZenML Cloud and gain access to a managed ZenML server instance, called a tenant, with exciting new features.

<figure><img src="../../.gitbook/assets/Screenshot 2023-07-28 at 15.45.04.png" alt=""><figcaption></figcaption></figure>

## Step 1: Sign-Up

Visit [the ZenML website](https://www.zenml.io/home) and navigate to the ZenML Cloud sign-up page. You can find it in the top-right corner.

On the ZenML Cloud sign-up page, you will have the option to sign up using either your Google account or your GitHub account. Click on the respective button to proceed with your preferred sign-up method.

After successfully signing in with your Google or GitHub account, you will be asked to complete your ZenML Cloud profile. Provide any additional information required to set it up, such as your name, email address, and organization details.

<figure><img src="../../.gitbook/assets/Screenshot 2023-07-28 at 15.45.55.png" alt=""><figcaption></figcaption></figure>

## Step 2: Automatic tenant creation

> Upon completing the sign-up process, you will gain access to a free trial period lasting for 30 days. During this trial, a ZenML tenant will be automatically created for you. This setup process typically takes a few minutes to complete, and once it's ready, you will receive an email notification. This will mark the beginning of your 30-day trial, allowing you to explore and experience the benefits of ZenML's features firsthand.

Reviewer: ```suggestion
Upon completing the sign-up process, you will gain access to a free trial period lasting for 30 days. You will have access to a ZenML tenant (automatically created for you behind the scenes) for the duration of this trial period. This setup process typically takes a few minutes to complete, and once it's ready you will receive an email notification. This will mark the beginning of your 30-day trial, allowing you to explore and experience the benefits of ZenML's features firsthand.
```
